CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 BUG FIXES:
 * `terraform test`: Stop attempting to destroy run blocks that have no actual infrastructure to destroy. This fixes an issue where attempts to destroy "verification" run blocks that load only data sources would fail if the underlying infrastructure referenced by the run blocks had already been destroyed. ([#34331](https://github.com/hashicorp/terraform/pull/34331))
+* `terraform plan -out`: prevent running saved cloud plans in VCS-connected workspaces, since applying configurations from the command line in these workspaces is almost never desirable.
```

Comment:
I tend to think the component should be `cloud` or `cloud save plans` but I leave it up to your judgement

Reply:
Oh, hmm. Lemme scroll the changelog and see what prior art says.

Reply:
You're right, it should be "cloud"!